### DISCUSSION TITLE
Could not find TameMyCerts.dll

### DISCUSSION THREAD

**Discussion by patboy29** [Sleepw4lker/TameMyCerts#31] [Q&A] [answered]
Just downloaded and trying to install but I get an error saying Could not find TameMyCerts.dll. This file is not in the repo. Do I have to rename one of the other .dll?

**Comment by hansjoachimknobloch** [comment]
Did you run `install.ps1`  from within the directory where you expanded the release zip-file?
It looks for `TameMyCerts.dll` in the current working directory.

**Comment by patboy29** [comment]
Yes but there is no TameMyCerts.dll in any folder expanded from the zip. The only dlls available are CERTCLILIB.dll and CERTPOLICYLIB.dll. Same has in the Github repo

**Comment by hansjoachimknobloch** [comment, marked as answer]
Could it be possible that you downloaded the source zip (that must be built first) and not the binary release zip under https://github.com/Sleepw4lker/TameMyCerts/releases/download/1.6.1045.1129/TameMyCerts_community_1.6.1045.1129.zip ?

**Reply by patboy29** [reply]
My bad, thanks for pointing me to the right place

**Reply by hansjoachimknobloch** [reply]
You're welcome. I am a fan of Uwe's great work here and always happy to see people using it.
